ui/imports/shared/popups/AccountsModalHeader.qml

```diff
@@ -36,9 +30,9 @@ StatusComboBox {
         width: contentItem.childrenRect.width + control.leftPadding + control.rightPadding
         height: 32
         radius: 8
-        color: !!selectedAccount ? hoverHandler.containsMouse ?
-                                       Theme.palette.walletAccountColors.getHoveredColor(selectedAccount.color) :
-                                       selectedAccount.color ?? "transparent" : "transparent"
+        color: !!selectedAccount ? hoverHandler.hovered ?
+                                       Utils.getHoveredColor(selectedAccount.colorId) :
+                                       Utils.getColorForId(selectedAccount.colorId) ?? "transparent" : "transparent"
```

Comment:
This `Utils.getColorForId(selectedAccount.colorId) ?? "transparent"` will always be true, never false.